Source/WarpX.H

```diff
@@ -243,6 +243,7 @@ public:
     amrex::MultiFab * get_pointer_F_fp  (int lev) const { return F_fp[lev].get(); }
     amrex::MultiFab * get_pointer_G_fp  (int lev) const { return G_fp[lev].get(); }
     amrex::MultiFab * get_pointer_phi_fp  (int lev) const { return phi_fp[lev].get(); }
+    amrex::MultiFab * get_pointer_full_phi_fp (int lev) const {return full_phi_fp[lev].get();}
```

Comment:
I think it's better to use these `get_pointer_...` functions and also add a `get_pointer_full_rho_fp` function rather than directly accessing the multifab objects, since these return a std::unique_ptr which automatically performs cleanup on itself when it goes out of scope, and seems to be a good practice to use.